<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pigFarm</class>
 <widget class="QDialog" name="pigFarm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="btnPig1">
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>51</height>
      </size>
     </property>
     <property name="text">
      <string>猪栏1</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="btnPig2">
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>51</height>
      </size>
     </property>
     <property name="text">
      <string>猪栏2</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btnPig3">
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>51</height>
      </size>
     </property>
     <property name="text">
      <string>猪栏3</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnPig4">
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>51</height>
      </size>
     </property>
     <property name="text">
      <string>猪栏4</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnPig5">
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>51</height>
      </size>
     </property>
     <property name="text">
      <string>猪栏5</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnPig6">
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>51</height>
      </size>
     </property>
     <property name="text">
      <string>猪栏6</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
